### PR TITLE
Config Update: Delete ticket on close button

### PR DESCRIPTION
**Pull request by Uo1428**
config/config.jsonc
Screenshot 2024-06-15 191547

Issue: #336

### CODE REVIEW

File: src/structure/ExtendedClient.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import {Client, ClientOptions, Collection, Routes} from "discord.js";
 import {BaseCommand, ConfigType} from "./";
-import {PrismaClient} from "@prisma/client";
+import {PrismaClient} from "../../node_modules/.prisma/client";
```
Comment (Sayrix) [inline src/structure/ExtendedClient.ts:3]: Why have you changed the path ?

**Review by Sayrix — CHANGES_REQUESTED** [review]

**Comment by Sayrix** [comment]
Also, can you describe what happens when deleteTicket is true and that you close the ticket ?

File: src/structure/ExtendedClient.ts (hunk 1 above)
Comment (Uo1428) [reply src/structure/ExtendedClient.ts:3]: because i got this error `Module '"@prisma/client"' has no exported member 'PrismaClient'.ts(2305)`

Comment (Sayrix) [reply src/structure/ExtendedClient.ts:3]: You had to do `npx prisma generate`

Comment (Uo1428) [reply src/structure/ExtendedClient.ts]: yes i see

**Comment by Uo1428** [comment]
First, it will close the ticket as before. Then, within 15 seconds, the ticket will be deleted.

**Comment by Sayrix** [comment]
So does it generate a transcript and give it in dms ect ?

**Comment by Uo1428** [comment]
yes

File: src/events/interactionCreate.ts [hunk 2]
```diff
@@ -127,11 +140,11 @@ export default class InteractionCreateEvent extends BaseEvent {
 
 			if (interaction.customId === "close") {
 				await interaction.deferReply({ ephemeral: true }).catch((e) => console.log(e));
-				close(interaction, this.client, this.client.locales.getSubValue("other", "noReasonGiven"));
+				close(interaction, this.client, this.client.locales.getSubValue("other", "noReasonGiven"), this.client.config.closeOption.deleteTicket);
```
Comment (zhiyan114) [inline src/events/interactionCreate.ts:143]: T

**Review by Sayrix — APPROVED** [review]

**Pull request merged**